Client/trunk/ParaEngineClient/Core/ParaEngineAppBase.cpp

```diff
@@ -834,6 +834,20 @@ bool CParaEngineAppBase::FindParaEngineDirectory(const char* sHint)
 		::GetCurrentDirectory(MAX_PATH, sWorkingDir);
 #endif
 		m_sInitialWorkingDir = sWorkingDir;
+		if (m_sInitialWorkingDir.find("\\bin64") != std::string::npos || m_sInitialWorkingDir.find("/bin64") != std::string::npos) {
```

Comment:
is this necessary? we can use paraengine.sig to locate working directory.both 32 and 64 should use the same working directory.